packages/core/src/schema/models/providerLogs.ts

```diff
@@ -24,6 +23,7 @@ export const logSourcesEnum = latitudeSchema.enum('log_source', [
 export const providerLogs = latitudeSchema.table('provider_logs', {
   id: bigserial('id', { mode: 'number' }).notNull().primaryKey(),
   uuid: uuid('uuid').notNull().unique(),
+  documentLogUuid: uuid('document_log_uuid'),
```

Comment:
In provider_logs table document UUID is nullable. We might want to have provider logs without a linked document log. 

Please confirm this assumption is true